pkgs/applications/networking/mailreaders/neomutt/default.nix

```
@@ -6,14 +6,14 @@
 }:
 
 stdenv.mkDerivation rec {
-  version = "20231221";
+  version = "20240201";
   pname = "neomutt";
 
   src = fetchFromGitHub {
     owner  = "neomutt";
     repo   = "neomutt";
     rev    = version;
-    sha256 = "sha256-IXly2N/DD2+XBXVIXJw1sE/0eJwbUaONDNRMi7n1T44=";
+    sha256 = "sha256-1yOjS2uHY66cKhUB9nwnRUpfg9VK5uWU9TcTXbX7shU=";
```

Minor nit:

```suggestion
    hash = "sha256-1yOjS2uHY66cKhUB9nwnRUpfg9VK5uWU9TcTXbX7shU=";
```

ℹ️ The same change could be made for the `test-file` and `contrib` [below](https://github.com/NixOS/nixpkgs/pull/285578/files#diff-f6b0ff192b29c6d77d458a8d627b07d98bccb5088b959211e883cabc8da788e9R88-R101).